ShemaM/My-Brand
# Ch classes

#### What does this PR do?
> using classes in controllers instead of using pure function
#### Description of Task to be completed
> turned all controller functions into one class `postControllers`
#### How should this be manually tested?
> clone this repo by `https://github.com/ShemaM/My-Brand.git`
> git checkout `ch-classes`
> go check on folder `controllers`
> go to `post.js`
